code/game/objects/items/rogueitems/natural/wood.dm

```diff
@@ -56,38 +80,26 @@
 	smeltresult = /obj/item/rogueore/coal
 	lumber_amount = 0
 	metalizer_result = /obj/item/rogueore/copper
+/obj/item/grown/log/tree/small/attackby(obj/item/I, mob/living/user, params)
+	user.changeNext_move(CLICK_CD_MELEE)
+	var/skill_level = user.mind.get_skill_level(/datum/skill/labor/lumberjacking)
+	var/planking_time = (45 - (skill_level * 5))
```

Comment:
the current grind stone is base 40 with skill level minus 5 per skill level, shouldnt the item it self match at 40 or be even lower at 35 because you have to go out of your way to get the custom tool for it instead of the public grind stones that already exist?